movieapp/src/actions/actionTypes.ts

```diff
@@ -0,0 +1,3 @@
+export const RECEIVE_MOVIES_FROM_API_SUCCESS = 'RECEIVE_MOVIES_FROM_API_SUCCESS';
+export const RECEIVE_MOVIES_FROM_API_ERROR = 'RECEIVE_MOVIES_FROM_API_ERROR';
+export const SORT_MOVIES_BY_ID = 'SORT_MOVIES_BY_ID';
```

Comment:
it's possible not to create these constants

with npm module called `redux-actions`